docs/api/RESTAPI.md

```diff
@@ -554,6 +556,48 @@ Retrieve the outputs for the specified workflow. Cromwell will return any output
 |**oauth2**|**[google_oauth](#google_oauth)**|openid|
 
 
+<a name="api-workflows-version-id-releasehold-post"></a>
+## Switch a workflow from 'On Hold' to 'Submitted' status
+```
+POST /api/workflows/{version}/{id}/releaseHold
+```
+
+
+#### Description
+Request Cromwell to release the hold on a workflow. It will switch the status of a workflow from 'On Hold' to 'Submitted' so it can be picked for running. For instance this might be necessary in cases where you have submitted a workflow with workflowOnHold = true.
+
+
+#### Parameters
+
+|Type|Name|Description|Schema|Default|
+|---|---|---|---|---|
+|**Path**|**id**  <br>*required*|A workflow ID|string||
+|**Path**|**version**  <br>*required*|Cromwell API Version|string|`"v1"`|
+
+
+#### Responses
+
+|HTTP Code|Description|Schema|
+|---|---|---|
+|**200**|Successful Request|[WorkflowIdAndStatus](#workflowidandstatus)|
+|**400**|Malformed Workflow ID|No Content|
+|**403**|Workflow in terminal status|No Content|
```

Comment:
Is this true ? I think it's returning 400 now if the workflow is not `onHold`
Returning 403 might actually be better

Reply:
Makes sense. Changed it to 403 and pushed the changes.